backends/nxp/backend/ir/converter/node_converters/ops_converters/convolution_converter.py

```diff
@@ -79,8 +79,8 @@ def _is_supported_on_target(
                         return False
                 elif conv_utils.group_conv_convertible_into_multiple_convolutions(
                     node, groups
-                ):  # Separable conv.
-                    # Requires addition of `Split` and `Concatenation` operators, which are not supported on Neutron.
+                ):  # Separable conv. This should never be reached, as the node should have been decomposed into
+                    #  multiple parallel convolutions by the `SplitGroupConvolution` pre-processing pass.
```

Comment:
Add a `logging.warning` here, in order to help to backtrack the problem, as later on in `_convert_2d_conv` it will raise a runtime error.